src/carriers/xmlrpc_carrier/XmlRpcStream.cpp

```diff
@@ -129,8 +129,8 @@ yarp::conf::ssize_t XmlRpcStream::read(Bytes& b)
             if (ok) {
                 //printf("got a block!\n");
                 XmlRpcValue xresult;
-                std::string prefix = "";
-                std::string cprefix = "";
+                std::string prefix;
+                std::string cprefix;
```

Comment:
because std::string is default initialized as an empty string.
see https://clang.llvm.org/extra/clang-tidy/checks/readability-redundant-string-init.html